module-presentation/src/main/kotlin/site/yourevents/invitation/dto/response/InvitationInfoResponse.kt

```diff
@@ -0,0 +1,31 @@
+package site.yourevents.invitation.dto.response
+
+import site.yourevents.invitation.domain.Invitation
+import site.yourevents.invitationinformation.domain.InvitationInformation
+import site.yourevents.invitationthumnail.domain.InvitationThumbnail
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class InvitationInfoResponse(
+    val invitationId: UUID,
+    var createDate: LocalDateTime,
+    var title: String,
+    var schedule: LocalDateTime,
+    var location: String,
+    var url: String,
```

Comment:
클라이언트 측에서 어떤 url인지 한눈에 알아보기 어려울 것 같다는 생각입니다!!
다음과 같이 수정하면 좋을 것 같습니다!
```suggestion
    var thumbnailUrl: String,
```

Reply:
수정했습니다!